test-curves/src/bls12_381/mod.rs

```diff
@@ -22,12 +24,16 @@ pub mod g2_swu_iso;
 pub use {fq::*, fq12::*, fq2::*, fq6::*, g1::*, g1_swu_iso::*, g2::*, g2_swu_iso::*};
 
 #[cfg(test)]
+#[cfg(feature = "bls12_381_curve")]
 mod tests;
 
+#[cfg(feature = "bls12_381_curve")]
 pub type Bls12_381 = Bls12<Parameters>;
 
+#[cfg(feature = "bls12_381_curve")]
 pub struct Parameters;
 
+#[cfg(feature = "bls12_381_curve")]
 impl Bls12Parameters for Parameters {
```

Comment:
The main reason for this change is that `Parameters` requires group definitions.